workshops/templates/forms/trainingrequest.html

```diff
@@ -57,6 +58,11 @@
 <p>For more information on Software and Data Carpentry instructor training,
 please see <a href="http://swcarpentry.github.io/instructor-training/" target="_blank">
 http://swcarpentry.github.io/instructor-training/</a>.</p>
+
+<div class="alert alert-warning">
+  <p><strong>Warning:</strong> Applications for the open Fall 2016 online instructor training class are now <strong>closed</strong>, but we are still accepting applications for subsequent courses. If you are scheduled to receive training at a partner site, and have been asked to fill in this form to give us information about yourself, please do so.</p>
```

Comment:
1. Review would be easier if you line-wrapped :-)
2. Please remove the word "Warning", and the bold around "closed".
3. The rest looks good to me - @maneesha, can you please review the wording?